test/sql/types.result

```diff
@@ -1378,7 +1378,7 @@ box.execute("INSERT INTO t VALUES (1, x'616263');")
 ...
 box.execute("ALTER TABLE t DROP CONSTRAINT fk1;")
 ---
-- row_count: 1
+- row_count: 0
```

Comment:
Is it documented anywhere what row_count values our DDL operations are supposed to return? I have a suspicion that we specifically wanted to return `1` for any DDL except `0` for cases when `IF EXISTS` / `IF NOT EXIST` are used and fail.

Reply:
Fixed. For DROP CONSTRAINT I fixed in #7651.